mobile/native/src/state.rs

```diff
@@ -51,6 +52,18 @@ pub fn get_node() -> Arc<Node> {
     NODE.get().read().clone()
 }
 
+pub fn register_pending_usdp_invoices(hash: bitcoin::hashes::sha256::Hash) {
```

Comment:
🔧 shouldn't these functions rather be on the node than in the `state.rs`? 

This file should only contain boilerplate code for getting and setting states. (Storage)

Reply:
done